hack/pin-deps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Pin all k8s.io dependencies to a specified version.
+#
+
+VERSION=1.12.5
```

Comment:
to pin to new k8s version, we can upgrade this VARIABLE and run this script to pin dependencies